### PR TITLE
Clarify working group lead requirements

### PR DESCRIPTION
**Pull request by craigbox**
Harmonize this doc with https://github.com/istio/community/ROLES.md.

### CODE REVIEW

File: WORKING-GROUP-PROCESSES.md [hunk 1]
```diff
@@ -87,10 +87,9 @@ technical oversight committee also reserves the right to dissolve or recharter w
 
 ## Leads
 
-Each working group must have 2 or 3 leads, with at least two different organizations represented. Working group leads must be members of the
-Istio project (that is, have made multiple contributions to the project in the form of code, design, or documentation).
+Each working group must have 2 or 3 leads, with at least two different organizations represented. Working group leads must meet certain requirements, including having been members of the Istio project for a year, and a maintainer for at least 3 months.
```
Comment (linsun) [inline WORKING-GROUP-PROCESSES.md:90]: Is 1 year and 3 month the hard criterias?  We are not strictly following that with the new networking wG lead or security WG lead.

Comment (howardjohn) [reply WORKING-GROUP-PROCESSES.md:90]: We also did not follow 1 year in a few cases

Comment (smawson) [reply WORKING-GROUP-PROCESSES.md:90]: I think we're allowed to make exceptions, maybe we should mention that we can make exceptions? These are the ideal requirements...

Comment (dcberg) [reply WORKING-GROUP-PROCESSES.md:90]: If we commonly use exceptions then we should consider softening the requirements. Maybe 6 months membership and 1 month maintainer as an example.

Comment (dcberg) [reply WORKING-GROUP-PROCESSES.md:90]: At the very least the document should be clear that exceptions are at the discretion of the TOC. The doc does not make this point clear.

**Pull request merged**